src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -175,79 +177,91 @@
+         {:keys [createdBy createdDate bucketName description tags workspace-attributes library-attributes]} :workspace} workspace
+        render-detail-box (fn [title & children]
+                            [:div
+                             {:style {:flexBasis "50%" :paddingRight "2rem" :marginBottom "2rem"}}
+                             [:div {:style {:paddingBottom "0.5rem"}}
+                              (style/create-section-header title)]
+                             (map-indexed
+                              (fn [i child]
+                                (if (even? i)
+                                  [:div {:style {:fontWeight 500 :paddingTop "0.5rem"}} child]
+                                  [:div {:style {:fontSize "90%" :lineHeight 1.5}} child]))
+                              children)])
         processed-tags (flatten (map :items (vals tags)))]
     [:div {:style {:flex "1 1 auto" :overflow "hidden"}}
-     [:div {:style {:display "flex" :flexWrap "wrap"}}
-      (render-detail-box
-        1
-        (str "Workspace Owner" (when (> (count owners) 1) "s"))
-        (style/create-paragraph
-          [:div {}
-           (interpose ", " owners)]))
-      (render-detail-box
-        3
-        "Created By"
-        (style/create-paragraph
-          [:div {} createdBy]
-          [:div {} (common/format-date createdDate)]))
+     [:div {:style {:display "flex"}}
       (render-detail-box
-        2
-        "Google Bucket"
-        (style/create-paragraph
-          (case bucket-access?
-            nil [:div {:style {:position "absolute" :marginTop "-1.5em"}}
-                 [comps/Spinner {:height "1.5ex"}]]
-            true (style/create-link {:text bucketName
-                                     :href (str moncommon/google-cloud-context bucketName "/")
-                                     :style {:color "-webkit-link" :textDecoration "underline"}
-                                     :title "Click to open the Google Cloud Storage browser for this bucket"
-                                     :target "_blank"})
-            false bucketName)
-          (when (not reader?)
-            [:div {}
-             [:div {} (str "Total Estimated Storage Fee per month = " storage-cost)]
-             [:div {:style {:fontSize "80%"}} (str "Note: the billing account associated with " (:namespace workspace-id) " will be charged.")]])))
-      (render-detail-box
-        4
-        "Analysis Submissions"
-        (style/create-paragraph
-          (let [count-all (apply + (vals submissions-count))]
-            [:div {}
-             (str count-all " Submission" (when-not (= 1 count-all) "s"))
-             (when (pos? count-all)
-               [:ul {:style {:marginTop 0}}
-                (for [[status subs] (sort submissions-count)]
-                  [:li {} (str subs " " status)])])])))
+       "Workspace Access"
+
+       "Access Level"
+       (style/prettify-access-level user-access-level)
```

Comment:
Another option would be to have your map function work on pairs, e.g.:
```cljs
(map (fn [[sub-title content]] ...))
```
and pass these in pairs, e.g.:
```cljs
["Access Level"
 (style/prettify-access-level user-access-level)]
```
so the grouping is a bit more explicit and eliminate the need to add vertical space.